Dockerfile

```diff
@@ -0,0 +1,42 @@
+# Use the official Golang image to build the Go binary
+FROM golang:1.22 AS builder
+
+# Set the Current Working Directory inside the container
+WORKDIR /app
+
+# Copy go.mod and go.sum files and download dependencies
+COPY go.mod go.sum ./
+RUN go mod download
+
+# Copy the source code into the container
+COPY . .
+
+# Build the Go app
+RUN GOOS=linux GOARCH=amd64 go build -o main cmd/productivite/main.go
+
+# Use a minimal base image to run the Go binary
+FROM alpine:latest
```

Comment:
<div>

<strong>Bito Code Review Agent Run</strong> <a href=https://github.com/aakashgusain1990/productivite/pull/12#issuecomment-2426343786>#df96d7</a><span> - 10/21/2024, 04:38 pm</span>
<br>

<div><small>🔴</small> High importance</div>

<div id="suggestion">
<div id="issue">
<b>Issue:</b> The Alpine base image is using the 'latest' tag, which can lead to inconsistent builds and potential security vulnerabilities.
</div>
<div id="fix">
<b>Fix:</b> Specify a precise version for the Alpine base image to ensure consistent and secure builds.
</div>
<details>
<summary>
<b>Code suggestion</b>
</summary>
<div id="code">

```
 @@ -15,4 +15,4 @@
  RUN GOOS=linux GOARCH=amd64 go build -o main cmd/productivite/main.go
 
  # Use a minimal base image to run the Go binary
 -FROM alpine:latest
 +FROM alpine:3.18
```
</div>
</details>
</div>
<br>


</div>

Is this a valid issue, or was it incorrectly flagged by the Agent?
- [ ] it was incorrectly flagged